documentation/src/main/markdown/releasenotes.md

```diff
@@ -1,5 +1,11 @@
 # Release notes
 
+## Version 8.6.0
+
+### Changed features
+
+* Signature scans will now wait for all scans that the scan CLI might invoke, such as snippet and string search scans, and not just signature scans.
```

Comment:
Suggested tweak: 
Signature scans, when invoked from the CLI, will now wait for all scans, such as snippet and string search, to complete before parsing Scan IDs.